src/helperFunctions/toast.ts

```diff
@@ -0,0 +1,21 @@
+import { toast } from 'react-toastify';
```

Comment:
Let's make this as `import { toast as toastify } from ....` and rename our function name to `toast`